### PR TITLE
Remove LogLevel UI for Xamarin apps

### PR DESCRIPTION
**Pull request by Cloudwu1**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
Remove log level UI for Xamarin apps.

### CODE REVIEW

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml.cs [hunk 1]
```diff
@@ -14,34 +11,9 @@ namespace Contoso.Forms.Demo
     public partial class AppCenterContentPage : ContentPage
     {
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs [hunk 2]
```diff
@@ -14,68 +11,25 @@ namespace Contoso.Forms.Puppet
     public partial class AppCenterContentPage : ContentPage
     {
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs [hunk 3]
```diff
@@ -14,68 +11,25 @@ namespace Contoso.Forms.Puppet
...
-            UpdateLogWriteLevelLabel();
-
-
             if (Xamarin.Forms.Device.RuntimePlatform == Xamarin.Forms.Device.iOS)
             {
                 Icon = "bolt.png";
             }
 
-            // Setup auth type dropdown choices
+            // Setup start type dropdown choices
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs]:
```suggestion
            // Setup start type dropdown choices.
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs [hunk 4]
```diff
@@ -126,9 +66,5 @@ async void UpdateEnabled(object sender, ToggledEventArgs e)
             await AppCenter.SetEnabledAsync(e.Value);
         }
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml [hunk 5]
```diff
@@ -2,22 +2,22 @@
...
-            <SwitchCell Text="Distribute Enabled" On="true" x:Name="DistributeEnabledSwitchCell" OnChanged="UpdateDistributeEnabled" />
+            <SwitchCell Text="Distribute Enabled" On="true" x:Name="DistributeEnabledSwitchCell" OnChanged="UpdateDistributeEnabled" OnColor="{StaticResource SwitchCellOnColor}" />
         </TableSection>
         <TableSection Title="Push">
-            <SwitchCell Text="Push Enabled" On="true" x:Name="PushEnabledSwitchCell" OnChanged="UpdatePushEnabled" />
+            <SwitchCell Text="Push Enabled" On="true" x:Name="PushEnabledSwitchCell" OnChanged="UpdatePushEnabled" OnColor="{StaticResource SwitchCellOnColor}"/>
         </TableSection>
         <TableSection Title="Auth">
-            <SwitchCell Text="Auth Enabled" On="true" x:Name="AuthEnabledSwitchCell" OnChanged="UpdateAuthEnabled" />
+            <SwitchCell Text="Auth Enabled" On="true" x:Name="AuthEnabledSwitchCell" OnChanged="UpdateAuthEnabled"  OnColor="{StaticResource SwitchCellOnColor}"/>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml]:
```suggestion
            <SwitchCell Text="Auth Enabled" On="true" x:Name="AuthEnabledSwitchCell" OnChanged="UpdateAuthEnabled" OnColor="{StaticResource SwitchCellOnColor}"/>
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml [hunk 6]
```diff
@@ -2,22 +2,22 @@
...
             <ViewCell>
                 <Picker x:Name="AuthTypePicker" Title="Auth Type" PropertyChanged="ChangeAuthType"/>
             </ViewCell>
         </TableSection>
         <TableSection Title="Rum">
-            <SwitchCell Text="Rum Enabled" x:Name="RumEnabledSwitchCell" OnChanged="UpdateRumEnabled" />
+            <SwitchCell Text="Rum Enabled" x:Name="RumEnabledSwitchCell" OnChanged="UpdateRumEnabled"  OnColor="{StaticResource SwitchCellOnColor}"/>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml]:
```suggestion
            <SwitchCell Text="Rum Enabled" x:Name="RumEnabledSwitchCell" OnChanged="UpdateRumEnabled" OnColor="{StaticResource SwitchCellOnColor}"/>
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml [hunk 7]
```diff
@@ -2,22 +2,22 @@
...
         </TableSection>
         <TableSection Title="EventFilter">
-            <SwitchCell Text="EventFilter Enabled" x:Name="EventFilterEnabledSwitchCell" OnChanged="UpdateEventFilterEnabled" />
+            <SwitchCell Text="EventFilter Enabled" x:Name="EventFilterEnabledSwitchCell" OnChanged="UpdateEventFilterEnabled"  OnColor="{StaticResource SwitchCellOnColor}" />
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml]:
```suggestion
            <SwitchCell Text="EventFilter Enabled" x:Name="EventFilterEnabledSwitchCell" OnChanged="UpdateEventFilterEnabled" OnColor="{StaticResource SwitchCellOnColor}" />
```

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml [hunk 8]
```diff
@@ -3,31 +3,12 @@
...
-                <StackLayout Orientation="Horizontal" Margin="15,0,15,0">
-                    <Label Text="Log Level" HorizontalOptions="StartAndExpand" VerticalOptions="Center" />
-                    <Label Text="{x:Static local:Constants.Verbose}" HorizontalOptions="EndAndExpand" VerticalOptions="Center" TextColor="Gray" x:Name="LogWriteLevelLabel" />
-                </StackLayout>
-            </ViewCell>
-            <ViewCell Tapped="WriteLog">
-                <Button Text="Write Log" InputTransparent="true" Clicked="WriteLog" />
-            </ViewCell>
-        </TableSection>
         <TableSection Title="Logging">
```
Comment (guperrot) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml]:
```suggestion
         <TableSection Title="Misc">
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml [hunk 9]
```diff
@@ -2,32 +2,13 @@
...
-                <StackLayout Orientation="Horizontal" Margin="15,0,15,0">
-                    <Label Text="Log Level" HorizontalOptions="StartAndExpand" VerticalOptions="Center" />
-                    <Label Text="{x:Static local:Constants.Verbose}" HorizontalOptions="EndAndExpand" VerticalOptions="Center" TextColor="Gray" x:Name="LogWriteLevelLabel" />
-                </StackLayout>
-            </ViewCell>
-            <ViewCell Tapped="WriteLog">
-                <Button Text="Write Log" InputTransparent="true" Clicked="WriteLog" />
-            </ViewCell>
-        </TableSection>
         <TableSection Title="Logging">
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml]:
```suggestion
         <TableSection Title="Misc">
```

**Comment by guperrot** [comment]
Screen Shot 2019-11-14 at 12 11 50 PM
Can you verify the changes by running the app?

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AnalyticsContentPage.xaml [hunk 10]
```diff
@@ -5,7 +5,7 @@
              x:Class="Contoso.Forms.Puppet.AnalyticsContentPage">
     <TableView Intent="Form">
         <TableSection Title="Analytics Settings">
-            <SwitchCell Text="Analytics Enabled" On="true" x:Name="EnabledSwitchCell" OnChanged="UpdateEnabled" />
+            <SwitchCell Text="Analytics Enabled" On="true" x:Name="EnabledSwitchCell" OnChanged="UpdateEnabled" OnColor="{StaticResource SwitchCellOnColor}"/>
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AnalyticsContentPage.xaml]: Screen Shot 2019-11-15 at 11 37 53 AM

These color changes are out of scope for the work item and causing crashes.

Comment (Cloudwu1) [reply Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AnalyticsContentPage.xaml]: Remove these color changes.

Comment (guperrot) [reply Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AnalyticsContentPage.xaml]: It definitely crashes even after a git clean.
